snapcraft/storeapi/__init__.py

```diff
@@ -387,6 +399,63 @@ def snap_release(self, snap_name, revision, channels):
 
         return response_json
 
+    def generate_snap_build(self, account_info, snap_name,
```

Comment:
This method doesn't belong to SCA layer, IMO. It lands and sign an assertion on disk and never talk to the Store.